Kernel/Term.hpp

```diff
@@ -324,27 +354,29 @@ class TermList {
   // but it was *not* portable because the layout of the bitfield is not guaranteed
   //
   // now we use a manual bitfield, as follows
+#define MK_FIELD(typ, name, Name, NAME, start, size) \
```

Comment:
Can you help me understand why is the new stuff the same as the old? Or is it? 

@MichaelRawson , could you please take a quick look here too?